src/view/view-impl.cpp

```diff
@@ -81,12 +81,12 @@ void wf::wlr_view_t::handle_minimize_hint(wf::surface_interface_t *relative_to,
     box.width = hint.width;
     box.height = hint.height;
 
-    this->minimize_hint = box;
+    view_impl->minimize_hint = box;
 }
 
 wlr_box wf::wlr_view_t::get_minimize_hint()
```

Comment:
since this function's real implementation has been moved to view_interface_t, you can remove the overridden function in wlr_view_t.

Reply:
```
wlr_box wf::wlr_view_t::get_minimize_hint()
{
    return view_impl->minimize_hint;
}
```
You mean this right?